tests/unit/helpers/courses.test.js

```diff
@@ -1120,7 +1131,7 @@ describe('updateCourse', () => {
     courseFindOneAndUpdate.restore();
   });
 
-  it('should update an intra course', async () => {
+  it('should update a field an intra course', async () => {
```

Comment:
il manque un mot ^^ tu pourrais rajouter of an intra course

Reply:
done